llama-index-integrations/llms/llama-index-llms-bedrock-converse/llama_index/llms/bedrock_converse/utils.py

```diff
@@ -16,14 +16,56 @@
 HUMAN_PREFIX = "\n\nHuman:"
 ASSISTANT_PREFIX = "\n\nAssistant:"
 
-FUNCTION_CALLING_MODELS = {
+BEDROCK_MODELS = {
+    "amazon.titan-text-express-v1": 8192,
+    "amazon.titan-text-lite-v1": 4096,
+    "amazon.titan-text-premier-v1:0": 3072,
+    "anthropic.claude-instant-v1": 100000,
+    "anthropic.claude-v1": 100000,
+    "anthropic.claude-v2": 100000,
+    "anthropic.claude-v2:1": 200000,
     "anthropic.claude-3-sonnet-20240229-v1:0": 200000,
     "anthropic.claude-3-haiku-20240307-v1:0": 200000,
     "anthropic.claude-3-opus-20240229-v1:0": 200000,
```

Comment:
One more model came out today, Claude Sonnet 3.5, that is SOTA!

I raised a new feature request: https://github.com/run-llama/llama_index/issues/14284

BTW, I was able to successfully use Bedrock Converse with Claude Sonnet 3 via this PR and it's more precise and hallucinating less than the plain ReAct Agent w/o tool use so this has been a great addition.